Digital-Circuits/README.md

```diff
@@ -1,2 +1,8 @@
 # Digital Circuits
 Please add any digital circuit in this folder in a separate .md file after following the main README.md.
+
+## Digital Logic Circuits
+
+Information regarding various types of Logic circuits used in electronic devices, this includes:
+- Combinational Logic Circuits
```

Comment:
Link the README.md to the exact .md file for ease of use
for eg. [Combinational Logic Circuits](/Digital-Logic-Circuits/Combinational-Logic-Circuit.md)